src/components/views/dialogs/UserPickerDialog.js

```diff
@@ -311,7 +268,7 @@ module.exports = React.createClass({
         // This is important, otherwise there's no way to invite
         // a perfectly valid address if there are close matches.
         const addrType = getAddressType(query);
-        if (addrType !== null) {
+        if (this.props.validAddressTypes.indexOf(addrType) !== -1) {
```

Comment:
I've been sticking to indexOf, but probably for no reason really, so I don't care, done